numpy/lib/__init__.py

```diff
@@ -44,11 +44,13 @@
 # numpy.lib namespace members
 from ._arrayterator_impl import Arrayterator
 from ._version import NumpyVersion
+from ._index_tricks_impl import ndindex
+ndindex.__module__ = "numpy.lib"
 
 __all__ = [
     "Arrayterator", "add_docstring", "add_newdoc", "array_utils",
     "format", "introspect", "mixins", "NumpyVersion", "npyio", "scimath",
-    "stride_tricks", "tracemalloc_domain",
+    "stride_tricks", "tracemalloc_domain", "ndindex",
```

Comment:
The addition of `ndindex` is making the CI fail. Either remove it here, or add it to https://github.com/numpy/numpy/blob/main/numpy/tests/test_public_api.py.

(Since `ndindex` was not present before in the namespace, my first reaction would be to keep it out)